apps/blunt_absinthe/lib/blunt/absinthe/absinthe_errors.ex

```diff
@@ -27,7 +27,13 @@ defmodule Blunt.Absinthe.AbsintheErrors do
         Enum.map(messages, fn message -> [message: message] end) ++ acc
 
       {key, messages}, acc when is_list(messages) or is_map(messages) ->
-        Enum.map(messages, fn message -> [message: "#{key} #{message}"] end) ++ acc
+        Enum.map(messages, fn
+          {field, messages} ->
+            [message: "#{key}.#{field} #{messages |> Enum.join(", ")}"]
+
+          message ->
+            [message: "#{key} #{message}"]
+        end) ++ acc
```

Comment:
I had a bit of trouble converting this to recursively call `format` to support more than one level deep. May need to switch to `Enum.flat_map` or `Enum.reduce` or mess with `++ acc`.